src/javascripts/components/showList/showList.js

```diff
@@ -1,36 +1,36 @@
-import showData from '../../helpers/data/showData';
 import showBuilder from '../showBuilder/showBuilder';
+import addShow from '../addShow/addShow';
+import showData from '../../helpers/data/showData';
 
-// eslint-disable-next-line import/no-cycle
-import authData from '../../helpers/data/authData';
 import utils from '../../helpers/utils';
 
-const showViewer = (e) => {
+const showViewEvent = (e) => {
   e.preventDefault();
-  console.error(e);
   $('#shows').removeClass('hide');
-  showData.getShows()
-    .then((shows) => {
-      let domString = `
-                    <div class="text-center">
-                      <h2 class="text-center">Upcoming Events</h2>
-                      <button class="btn btn-primary auth-button text-center" id="add-show">Add New Event</button>
-                    </div>
-                      <div class="display-flex flex-wrap myShows card-deck">`;
-      shows.forEach((show) => {
-        domString += showBuilder.showCardBuilder(show);
-      });
+  showBuilder.showCardBuilder();
```

Comment:
trying to think of a world where you would want to do the data call in here, then pass in `show` to the `showCardBuilder` inside of a `shows.forEach`, but I'm coming up empty 🙃